optuna/visualization/_regret_bound.py

```diff
@@ -0,0 +1,176 @@
+from __future__ import annotations
+
+from typing import NamedTuple
+
+import tqdm
+
+import optuna
+from optuna._experimental import experimental_func
+from optuna.logging import get_logger
+from optuna.study.study import Study
+from optuna.terminator.erroreval import BaseErrorEvaluator
+from optuna.terminator.erroreval import CrossValidationErrorEvaluator
+from optuna.terminator.improvement.evaluator import BaseImprovementEvaluator
+from optuna.terminator.improvement.evaluator import DEFAULT_MIN_N_TRIALS
+from optuna.terminator.improvement.evaluator import RegretBoundEvaluator
+from optuna.visualization._plotly_imports import _imports
+
+
+if _imports.is_successful():
+    from optuna.visualization._plotly_imports import go
+
+_logger = get_logger(__name__)
+
+
+PADDING_RATIO = 0.05
```

Comment:
[nit] I think the name, `PADDING_RATIO`, is too general to be put in this scope. How about `PADDING_RATIO_Y`?